cmds/boot/boot/boot.go

```diff
@@ -63,13 +67,45 @@ func updateBootCmdline(cl string) string {
 func main() {
 	flag.Parse()
 
+	if *verbose {
+		block.Debug = log.Printf
+	}
 	blockDevs, err := block.GetBlockDevices()
 	if err != nil {
 		log.Fatal("No available block devices to boot from")
 	}
 
 	// Try to only boot from "good" block devices.
 	blockDevs = blockDevs.FilterZeroSize()
+
+	// Parse and filter blocklist
+	if *blockList != "" {
+		pciList := pci.Devices{}
+		bL := strings.Split(*blockList, ",")
+		for _, b := range bL {
```

Comment:
can you move this whole parsing thing into a function we can call from internal diskboot?

Reply:
Done